### ISSUE TITLE
[.NET Core] The type or namespace name 'Event' could not be found

### ISSUE THREAD

**Issue by jchannon** [ical-org/ical.net#260]
I've recently upgraded from `2.3.1-xamarin-alpha` to `3.0.1-net-core-alpha` and get the above error.

The code in question is this:

```
        private string GetICalFile(Call call, string description, string name, int index)
        {

                var iCal = new Calendar();
                iCal.Scale = "GREGORIAN";
                iCal.Method = "PUBLISH";

                Event evt = iCal.Create<Event>();

                evt.Summary = call.Name + " " + call.ReferenceNo;
                evt.Description = description;
                evt.Start = new CalDateTime(call.UtcMeetingStartTime);
                evt.End = new CalDateTime(call.UtcStopTime);
                evt.Status = EventStatus.Confirmed;

                if (call.LauncherUserId != -1)
                {
                    evt.Organizer = new Organizer("MAILTO:" + call.LauncherUser.Email);
                }
                evt.Uid = call.ReferenceNo;

                var alarm = new Alarm
                {
                    Action = AlarmAction.Display,
                    Summary = "Call " + call.Name + " starting in 15 minutes.",
                    Trigger = new Trigger(TimeSpan.FromMinutes(-15))
                };
                evt.Alarms.Add(alarm);

                string fileName = Path.Combine(VQSystem.ICalFolder, call.ReferenceNo.Replace("/", "_") + name + "_" + index + ".ics");

                using (var fs = File.Open(fileName, FileMode.OpenOrCreate))
                {
                    var serializer = new CalendarSerializer();
                    serializer.Serialize(iCal, fs, Encoding.UTF8);
                }

                return fileName;

     }
```

Any ideas?

**Comment by jchannon** [comment]
Moving to `ical.Create<CalendarEvent>` allowed the code to compile however when serializing I got:

`"Object reference not set to an instance of an object."`

```
"at Ical.Net.Serialization.iCalendar.Serializers.Components.ComponentSerializer.<>c.<SerializeToString>b__6_0(ICalendarProperty p) in C:\\git\\ical.net\\net-core\\Ical.Net\\Ical.Net\\Serialization\\iCalendar\\Serializers\\Components\\ComponentSerializer.cs:line 40\n   
at System.Linq.EnumerableSorter`2.ComputeKeys(TElement[] elements, Int32 count)\n   at System.Linq.EnumerableSorter`1.ComputeMap(TElement[] elements, Int32 count)\n  
at System.Linq.EnumerableSorter`1.Sort(TElement[] elements, Int32 count)\n   
at System.Linq.OrderedEnumerable`1.ToList()\n   
at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\n   
at Ical.Net.Serialization.iCalendar.Serializers.Components.ComponentSerializer.SerializeToString(Object obj) in C:\\git\\ical.net\\net-core\\Ical.Net\\Ical.Net\\Serialization\\iCalendar\\Serializers\\Components\\ComponentSerializer.cs:line 40\n   
at Ical.Net.Serialization.iCalendar.Serializers.Components.ComponentSerializer.SerializeToString(Object obj) in C:\\git\\ical.net\\net-core\\Ical.Net\\Ical.Net\\Serialization\\iCalendar\\Serializers\\Components\\ComponentSerializer.cs:line 55\n   
at Ical.Net.Serialization.iCalendar.Serializers.SerializerBase.Serialize(Object obj, Stream stream, Encoding encoding) in C:\\git\\ical.net\\net-core\\Ical.Net\\Ical.Net\\Serialization\\iCalendar\\Serializers\\SerializerBase.cs:line 61\n   
at VQ.Classes.Alert.EmailAlertProcessor.GetICalFile(Call call, String description, String name, Int32 index) in /Users/jonathan/Projects/vq/src/VQClassFactory/Classes/Alert/EmailAlertProcessor.cs:line 174"
```

**Comment by rianjs** [comment]
@jchannon - is this still a bug? A bunch of changes made by you and others were merged today. One of them was getting the unit tests working for .NET Core, and they're all passing now.

**Comment by jchannon** [comment]
Fixed by #270

**Comment by rianjs** [comment]
Published in Ical.Net 3.0.9-net-core-beta:
https://www.nuget.org/packages/Ical.Net/
